src/cms/static/js/DeleteRowTable.js

```diff
@@ -0,0 +1,4 @@
+var x = document.getElementsByTagName("tr");
+for(i=5; i<= x.length; i++){
+    document.getElementById("myTable").deleteRow(5);
+}
```

Comment:
This file should be deleted, because it is not required anymore.

Reply:
I deleted this file